schema.graphql

```diff
@@ -28,16 +30,19 @@ type Wearer @entity {
 type Tree @entity {
   "Tree ID is its top hat domain - first 4 bytes of the top hat ID"
   id: ID!
+  linkedToTree: Tree
+  linkedToHat: Hat
   hats: [Hat!]! @derivedFrom(field: "tree")
+  linkedFromTrees: [Tree!]! @derivedFrom(field: "linkedToTree")
```

Comment:
I'm finding the "linkedTo" and "linkedFrom" labels a bit confusing. Maybe its just me being dense and I'll get used to it, but I'm wondering if "parent / child" semantics might be clearer. I'm not sold on this, but perhaps something like "childOfTree" instead of "linkedToTree", and "parentOfTrees" instead of "linkedFromTrees"? 

I could be getting those backwards, btw -- which I suppose is kind of my point 😉